wagtail/contrib/modeladmin/templates/modeladmin/includes/result_count.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
 {% block result_count %}
-    <span class="result-count">{% blocktrans %}{{ result_count }} out of {{ all_count }}{% endblocktrans %}</span>
-{% endblock %}
\ No newline at end of file
+    <span class="result-count">{% blocktrans trimmed %}{{ result_count }} out of {{ all_count }}{% endblocktrans %}</span>
+{% endblock %}
```

Comment:
Slight concern with the extra newline here but hopefully since there’s already one it collapses anyway.